src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int WORKING_DAY_DATE = 0;
+    private static final int WORKERS_NAME = 1;
+    private static final int WORK_TIME = 2;
+    private static final int PER_HOUR_SALARY = 3;
+    private DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromFormat = LocalDate.parse(dateFrom, dtf);
+        LocalDate dateToFormat = LocalDate.parse(dateTo, dtf);
+        StringBuilder result = new StringBuilder("Report for period ");
+        result.append(dateFrom).append(" - ").append(dateTo);
+        int salary = 0;
+        for (String name : names) {
+            for (String dataElement : data) {
```

Comment:
```suggestion
        for (String name : names) {
            int salary = 0;
            for (String dataElement : data) {
```

with such approach you won't need to reset variable manually